lib/src/main/java/io/ably/lib/transport/ITransport.java

```diff
@@ -51,6 +51,7 @@ public TransportParams() {
 
 		public Param[] getConnectParams(Param[] baseParams) {
 			List<Param> paramList = new ArrayList<Param>(Arrays.asList(baseParams));
+			paramList.add(new Param("v", HttpUtils.X_ABLY_VERSION_VALUE));
```

Comment:
Can we have a test for this please to prevent regressions?  I could have sworn this has been implemented previously though as I recall seeing tests, perhaps they only exist for REST?